### DISCUSSION TITLE
مشکل بسته شدن سابدامنه توسط کلادفلیر

### DISCUSSION THREAD

**Discussion by parsa-hassanloo** [hiddify/Hiddify-Manager#4579] [General سایر]
الا چند روز هستش هر سابدامنه‌ای که تو cloudflare ثبت کردم و تیک proxy هم که میزنم، بعد چند دقیقه وقتی میخوام به سابدامنه خودم وارد بشم (مثلا پنل فیلتر شکن خودم هستش) یه علامت قرمز ضربدر میاد و مینویسه: sorry, you have been blocked
هرچی هم سرچ میکنم چیزی بابتش پیدا نمیکنم تا این که از هوش مصنوعی پرسیدم ، راهنمایی کرد که شاید security از طرف cloudflare داره این کارو میکنه که رفتم دیدم ، بعله، داره هر ip که سعی میکنه سابدامنه منو جستجو کنه بلاک میکنه و با افتخار میگه: یه حمله ddos رو گرفتم، و ip منو هم میزنه.

کسی به این مشکل بر خورده که کمکم کنه باید چیکار کنم؟ تو اینترنت چیزی نمیتونم پیدا کنم اصلا. الا همه سابدامنه هارو مجبور شدم تیک proxy رو خاموش کنم مگرنه اصلا نمیزاره هیچی بهش وارد بشه. با هر اینترنتی هم بخوایید سعی کردم، با هر دیوایسی که بخوایید امتحان کردم.

**Comment by blue-farid** [comment]
اقا منم همین مشکل رو دارم

**Comment by hlperfume** [comment]


**Reply by blue-farid** [reply]
دمت گرم. ولی اصن چرا اخه :))

**Comment by hlperfume** [comment]


**Reply by blue-farid** [reply]
خب اونا هم من درجریان نیستم. علتش چی بود؟ راهی نیست بشه پیشگیری کرد؟ قبلا با فیلترچی ها میجنگیدیم الانم باید با کلودفلر بجنگیم
